tests/library/test_gen.py

```diff
@@ -18,11 +18,11 @@ def test_stop_char():
     lm += "Count to 10: 1, 2, 3, 4, 5, 6, 7, " + gen('text', stop=",")
     assert lm["text"] == "8"
 
-def test_stop_list_side_effect():
+def test_stop_list_side_effect(selected_model):
```

Comment:
This is actually a quick port of #695 so that I can make progress here. That PR should be merged first, and then this one updated.

Reply:
done